.circleci/config.yml

```diff
@@ -50,6 +50,16 @@ jobs:
         - ./chart-operator
         - ./architect
 
+  e2eTestChartBasicPR:
+    environment:
+      TEST_DIR: "integration/test/chart/basic"
+    <<: *e2eTest
+  
+  e2eTestChartBasicMaster:
+    environment:
+      TEST_DIR: "integration/test/chart/basic"
+    <<: *e2eTest
```

Comment:
Is it on purpose that `TESTED_VERSION: "current"` is missing? Like, how are the tests different now?

Reply:
Yes good to raise that. The chartconfig CR tests look up the WIP or active version from the release index.

The chart CR tests don't need that. The versioning is controlled by the `chart-operator.giantswarm.io/version` label which is set via the apiextensions chart.

Reply:
Is that how it will work later on everywhere with the version label?

Reply:
I don't get a differences between `e2eTestChartBasicPR` and `e2eTestChartBasicMaster`, is it only naming difference?

Reply:
Sorry, I got it

Reply:
@xh3b4sd All the chart CR tests will use the version label. The chartconfig CR tests will be deleted when we retire the chartconfig CRD. 

Does that answer your question?